.codecov.yml

```diff
@@ -0,0 +1,19 @@
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
```

Comment:
Hmm! This feels a bit low to me?

Reply:
I can up this in a follow up PR.  The View/Text components need more tests so it be best to get everything to the coverage levels we want before bumping them.